redirects/models.py

```diff
@@ -65,6 +66,18 @@ class Redirect(models.Model):
 
     class Meta:
         constraints = [
+            models.CheckConstraint(
+                check=~Q(old_url=""),
+                name="old_url_is_not_empty",
+            ),
+            models.CheckConstraint(
+                check=Q(old_url__endswith="/"),
+                name="old_url_endswith_slash",
+            ),
+            models.CheckConstraint(
+                check=Q(old_url__startswith="/"),
+                name="old_url_startswith_slash",
+            ),
```

Comment:
Is it any more efficient to put these into a single constraint with `&`'d `Q`s?

Reply:
Not sure about efficiency but definitely seems better to have them in one 👍

Reply:
Also added some tests specifically for the constraints that I'd missed